### ISSUE TITLE
typo

### ISSUE THREAD

**Issue by orens77** [opentradesolutions/opentrade#16]
https://github.com/opentradesolutions/opentrade/blob/07e1b42bbaadd277dbb7dffec1ee7be1f5f5c5e1/src/opentrade/connection.cc#L841

Should this read:  `  Send(out);`  ?

**Comment by orens77** [comment]
Also there are a few queries that don't get a response on lines 888, 890, 895, 897 

And for request "bod" (line 398) should the response be the first matching "bod"?  Or a list of them?

**Comment by open-trade** [comment]
yes it is a typo

**Comment by open-trade** [comment]
it send bod one by one

**Comment by orens77** [comment]
this breaks the REST interface one response per request

**Comment by open-trade** [comment]
you are right. rest is added after websocket, i did not realize this problem because all my tests are based on websocket. i will review all interface to make rest work well later
